### ISSUE TITLE
Fix accessibility issue on Profile page

### ISSUE THREAD

**Issue by Rizvida1** [CDH-Studio/UpSkill#1376]
**Describe the bug**
Fix accessibility issues with the Profile pages.

**Expected behavior**

- [ ] Axe extensions should show no more errors or structure issues.
- [x] App should be compatible with screen readers
